west.yml

```diff
@@ -198,7 +198,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: e8ccf3e4ad0d6ba80b468747682bde43ec97b435
+      revision: pull/427/head
```

Comment:
in the manifest commit message, the title should be more specific about what the change in the module is, and the body would be preferable not to describe with "npi" because that is an nxp term, users/contributors of zephyr project might be confused about what it means